virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -138,9 +138,7 @@ For more information, see xref:../../virt/install/preparing-cluster-for-virt.ado
 [id="virt-4-14-networking-new"]
 === Networking
 //CNV-18090 OVN-Kubernetes secondary network
-* You can connect a virtual machine (VM) to an xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#virt-connecting-vm-to-ovn-secondary-network[OVN-Kubernetes secondary network] by using the web console or the CLI.
-
-//[id="virt-4-14-storage-new"]
+* You can connect a virtual machine (VM) to an xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#virt-connecting-vm-to-ovn-secondary-network[OVN-Kubernetes secondary network] by using the web console or the CLI//[id="virt-4-14-storage-new"]
```

Comment:
I am not sure if you want every after the slashes. The published output looks odd https://65415--docspreview.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-14-release-notes#virt-4-14-networking-new

Reply:
Oh no! I'm fixing that right now. Thanks.